addon/mixins/pikaday.js

```diff
@@ -239,5 +250,17 @@ export default Mixin.create({
 
   _updateOptions() {
     this.get('pikaday').config(this.get('_options'));
+  },
+
+  verifyInvalidDate(element) {
+    if (this.get('clearInvalidDate')) {
```

Comment:
Probably don't need to check `clearInvalidDate` _inside_ `verifyInvalidDate` if we're checking it outside as well.

One or the other is fine; preferable outside.